distrobox-init

```diff
@@ -415,63 +415,74 @@ if [ "${upgrade}" -ne 0 ] ||
 		if ! apk add "${shell_pkg}"; then
 			shell_pkg="bash"
 		fi
-		deps="
+		if apk add wolfi-base; then
+			deps="
+				busybox
+				gnutar
+				man-db
+				mesa
+				posix-libc-utils
+				sudo-rs
+			"
+		elif apk add alpine-base; then
+			deps="
+				bash-completion
+				docs
+				gcompat
+				libc-utils
+				lsof
+				man-pages
+				mandoc
+				musl-utils
+				pinentry
+				sudo
+				tar
+				vte3
+				which
+				$(apk search -q mesa-dri)
+				$(apk search -q mesa-vulkan)
+			"
+		fi
+		deps="${deps}
```

Comment:
it woud be good to change `${deps}` to `${deps:-}` so that we have a default value in case the if/else before fails